main.cpp

```diff
@@ -1 +1,239 @@
+                        int random = rand() % list.size(); // randomise the word
+                        string word = list[random];
+                        printoptions();
+                        break;
+                    }
+                    else if (input == 'n' || input == 'N')
+                    {
+                        return 0;
+                    }
+                    else
+                    {
+                        cout << "Invalid option please type either 'y' or 'n'" << endl;
+                    }
+                }
+            }
+            break;
+        case 2: // when user decides to guess a word
+            cout << "WARNING: You lose 2 tries if you fail to guess the word.\nUse lowercase ONLY." << endl;
+            cin >> wordinput;
+            if (wordinput == word)
+            {
+                cout << "Congratulations! You won." << endl;
+                cout << "\nThe word is " << word << "." << endl;
+                input = ' '; // resets the char 'input'
+                cout << "Would you like to play again? (Y/N)" << endl;
+                while (input != 'y' || input != 'Y')
+                {
+                    cin >> input;
+                    if (input == 'y' || input == 'Y')
+                    {
+                        guesses = 0;
+                        guessedchar.clear();
+                        srand(time(NULL));
+                        int random = rand() % list.size(); // randomise the word
+                        string word = list[random - 1];
+                        printoptions();
+                        break;
+                    }
+                    else if (input == 'n' || input == 'N')
+                    {
+                        return 0;
+                    }
+                    else
+                    {
+                        cout << "Invalid option please type either 'y' or 'n'" << endl;
+                    }
+                }
+            }
+            else
+            {
+                cout << "\nCorrect...NOT. Try again." << endl;
+                guesses += 2;
+            }
+            break;
+        case 3: // when user wants to view all the guessed letter
+            if (guessedchar.size() == 0)
+            {
+                cout << "\nYou have not guessed any letters" << endl;
+            }
+            else
+            {
+                cout << "\nguessed letters: ";
+                copy(guessedchar.begin(),
+                     guessedchar.end(),
+                     ostream_iterator<char>(cout, ", "));
+                cout << endl;
+            }
+            break;
+        case 4:
+            printoptions();
+            break;
+        case 5: // when user quits the game
+            cout << "Quitting Game..." << endl;
+            cout << "Press any key to close the program" << endl;
+            cin.ignore();
+            cin.get();
+            return 0;
+            break;
```

Comment:
this break is unnecessary, since the return above will...well, return

Reply:
this is not the recommended way to terminate the game: your main loop should keep on looping by checking a boolean variable (eg. 'isGrameRunning'), which is set to true by default. Then, when you want to quit, you simply turn it to false.
Similarly, when it's game over, you change the value to false:
`if(guesses == 0) isGameRunning = false;`